### PR TITLE
Password should be 8 characters long  (KMS-3807)

### PR DESCRIPTION
**Pull request by adlearningmate**
Display the error message "Passwords must be at least 8 characters long" .

---
This change is [Reviewable](https://reviewable.io/reviews/opensalt/opensalt/445)

### CODE REVIEW

File: tests/acceptance/features/user/public_user/password_length_eight_char.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+Feature: Create new account password should be 8 characters long
+
+    @public-user @user @add-user
```
Comment (JosephLeon) [inline tests/acceptance/features/user/public_user/password_length_eight_char.feature]: Add unique identifier ex: "@7620-0007"

File: tests/acceptance/features/user/public_user/password_length_eight_char.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        And I follow "Sign up"
+
+        Then I should see "Create new account"
+        Then I fill in the "Email Address" with "user@opensalt.com"
+        Then I fill in the "Password" with "Password123!"
+        Then I fill in the "Confirm Password" with "password"
+        Then I click "Submit"
+        Then I should see "Passwords do not match"
+
+    @public-user @user @add-user
```
Comment (JosephLeon) [inline tests/acceptance/features/user/public_user/password_length_eight_char.feature]: Add unique identifier ex: "@7620-0007"

File: tests/acceptance/features/user/public_user/password_length_eight_char.feature [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        And I follow "Sign up"
+
+        Then I should see "Create new account"
+        Then I fill in the "Email Address" with "user@opensalt.com"
+        Then I fill in the "Password" with "password"
+        Then I fill in the "Confirm Password" with "password"
+        Then I click "Submit"
+        Then I should see "Password does not match required criteria"
+
+    @public-user @user @add-user
```
Comment (JosephLeon) [inline tests/acceptance/features/user/public_user/password_length_eight_char.feature]: Add unique identifier ex: "@7620-0007"

File: src/Controller/User/SignupController.php [hunk 4]
```diff
@@ -134,7 +134,7 @@ public function signupAction(Request $request)
                 if ('dev' === $this->kernelEnv) {
                     $form->addError(new FormError(get_class($e).': '.$e->getMessage()));
                 } else {
-                    $form->addError(new FormError('Sorry, an error occurred while creating your account.'));
+                    $form->addError(new FormError('Sorry, Passwords must be at least 8 characters long.'));
```
Comment (roverwolf) [inline src/Controller/User/SignupController.php]: This is not the only error that could occur, should check for the specific error.

AddUserCommand should have the assertion updated for the new requirement as well.

**Pull request merged**